integration_tests/src/lib.rs

```diff
@@ -2589,7 +2589,6 @@ mod test {
     async fn integration_test_guest_replace_many_downstairs() -> Result<()> {
         // Test using the guest layer to verify we can replace one
         // downstairs, but not another while the replace is active.
-        const BLOCK_SIZE: usize = 512;
```

Comment:
These are no longer needed?  Did new rust clippy find this?  Not sure how otherwise we would have been leaving them in.

Reply:
Yeah, (what I assume is) the new clippy found this.